tests/util/Makefile

```diff
@@ -0,0 +1,16 @@
+
+CC := gcc
+CFLAGS := -I../../include -I. \
+          -g -O3 -Wall
```

Comment:
I tested this, and it looks like the best option is to suppress this error in the test Makefile by adding:

- `-Wno-format`

I see what your saying about the compiler differences. That int length issue always sneaks up in the worst ways, haha.